bookkeeper-proto/src/main/proto/BookkeeperProtocol.proto

```diff
@@ -64,6 +64,7 @@ enum OperationType {
     GET_BOOKIE_INFO = 8;
     START_TLS = 9;
     FORCE_LEDGER = 10;
+    GET_LIST_OF_ENTRIES_OF_A_LEDGER = 11;
```

Comment:
will change this to GET_LIST_OF_ENTRIES_OF_LEDGER and all the corresponding names will be changed to GetListOfEntriesOfLedger

Reply:
fixed it

Reply:
already fixed it